src/tutorials/creating-a-pathogen-repo/creating-an-ingest-workflow.rst

```diff
@@ -0,0 +1,465 @@
+
+.. include:: ../../snippets/uncurated-ncbi-dataset.rst
+
+If the data looks sufficient for your pathogen, then use the NCBI Datasets source in your workflow by
+updating the ``ingest/defaults/config.yaml`` config file.
+
+1. Add your pathogen's NCBI taxonomy ID to the ``ncbi_taxon_id`` parameter
+2. If there are other NCBI Datasets fields you would like to include in the download, you can add them to the ``ncbi_datasets_fields`` parameter
+3. Skip to the :ref:`curation-steps`.
+
+NCBI Entrez
+-----------
+
+If your pathogen requires data from other fields not parsed by NCBI Datasets, then you will need to use
+the `NCBI Entrez <https://www.ncbi.nlm.nih.gov/books/NBK25501/>`_ tool to download all available data in a GenBank file.
+
+1. Add an Entrez search term to the ``entrez_search_term`` parameter in the ``ingest/defaults/config.yaml`` config file.
+
+2. Create a custom script to parse the GenBank file into a flat `JSON Lines/NDJSON format <https://jsonlines.org/>`_.
+   (We may provide an example script in the future, but this is currently not available.)
+
+3. Edit the ``parse_genbank_to_ndjson`` rule in ``ingest/rules/fetch_from_ncbi.smk`` to use the custom script.
+
+4. Switch the `Snakemake ruleorder <https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#handling-ambiguous-rules>`_
+   within the ``ingest/rules/fetch_from_ncbi.smk`` file.
+
+.. code-block::
+
+    ruleorder: format_ncbi_datasets_ndjson < parse_genbank_to_ndjson
+
+4. Make sure the ``field_map`` parameters in the config file are using the field names of your custom NDJSON output.
+
+.. _curation-steps:
+
+Curation steps
+==============
+
+After the public data is downloaded, the next part of the workflow runs a pipeline of data curation commands and scripts
+to format the metadata and sequences.
+
+The long term goal is to build out the :doc:`augur curate <augur:usage/cli/curate/index>`
+suite of commands to include all of the custom curation steps.
+For now, we've bundled custom scripts into the `ingest repository <https://github.com/nextstrain/ingest>`_ that is then
+vendored in the pathogen-repo-guide using `git-subrepo <https://github.com/ingydotnet/git-subrepo>`_.
+Please do not edit the vendored scripts in ``ingest/vendored`` directly.
+If you run into issues or encounter bugs with the vendored scripts, please `make an issue in the ingest repository <https://github.com/nextstrain/ingest/issues/new/choose>`_.
+Once the bug has been fixed in the original source code, you can follow the `instructions to update the vendored scripts <https://github.com/nextstrain/ingest?tab=readme-ov-file#vendoring>`_.
+
+We highly encourage you to go through the commands and custom scripts used in the ``curate`` rule within ``ingest/rules/curate.smk``
+to gain a deeper understanding of how they work.
+We will give a brief overview of each step and their relevant config parameters defined in ``ingest/defaults/config.yaml`` to help you get started.
```

Comment:
[discussion point, not request for changes]

Here we are mostly taking a template curate pipeline and describing the config values you can change. The other approach would be to detail the curate commands themselves and teach people how to add them to the bash command + abstract their parameters to the YAML config. This reminds me of a similar situation in visualisation where some people build libraries of charts while others expose the methods to build such charts (e.g. see Leland Wilkinson quote [here](https://medium.com/@mbostock/introducing-d3-scale-61980c51545f)).

Reply:
> The other approach would be to detail the curate commands themselves and teach people how to add them to the bash command + abstract their parameters to the YAML config. 

Hmm, that seems to be stepping into Snakemake tutorial territory...

Reply:
I guess I'm wondering how feasible it is for a user to develop an ingest pipeline _without_ knowing Snakemake. (This is the analogue of designing/documenting a charting library.) If that's going to work for a number of users then great! The phrasing you used here ("highly encourage you to go through the [snakemake rules]", "we will give you a brief overview" etc) made it seem to me like knowing Snakemake was a prerequisite. 

Anyway, the docs as they stand are an improvement so I wouldn't want this to hold up their merge. Rather something to think about as we move forward.

Reply:
> made it seem to me like knowing Snakemake was a prerequisite.

Yup! I have Snakemake listed as a prerequisite for the "Creating an ingest workflow" tutorial at the top:

https://github.com/nextstrain/docs.nextstrain.org/blob/79a0176a4a76e120ebefb42afa90fb073c7d46fc/src/tutorials/creating-a-pathogen-repo/creating-an-ingest-workflow.rst?plain=1#L22-L24